Update sql-api-sql-query.md

joelhulen opened the pull request:
Changed from "ends with" to "starts with" in the description for the STARTSWITH operator.

azurecla commented:
Hi __@SergeyKanzhelev__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

joelhulen committed:
Update sql-api-sql-query.md

Changed from "ends with" to "starts with" in the description for the STARTSWITH operator.
